commit 5557352a7793975bcaedf5f18b4e2c2883245f5b
Author: Akzhan Abdulin <akzhan.abdulin@gmail.com>
Date:   Mon Jan 17 22:20:47 2011 +0300

    Fix uninitialized constant RSpec::Core::ExampleGroup::Nested_2::Nested_1::YAML error

diff --git a/spec/cli/config_spec.rb b/spec/cli/config_spec.rb
index 1ad52f4..4a5288e 100644
--- a/spec/cli/config_spec.rb
+++ b/spec/cli/config_spec.rb
@@ -1,5 +1,7 @@
 require File.dirname(__FILE__) + '/../spec_helper'
 
+require 'yaml'
+
 describe YARD::CLI::Config do
   before do
     @config = YARD::CLI::Config.new
diff --git a/spec/config_spec.rb b/spec/config_spec.rb
index 6e578fc..316d7b8 100644
--- a/spec/config_spec.rb
+++ b/spec/config_spec.rb
@@ -1,5 +1,7 @@
 require File.join(File.dirname(__FILE__), "spec_helper")
 
+require 'yaml'
+
 describe YARD::Config do
   describe '.load' do
     before do
